src/lib/constants.js

```diff
@@ -11,9 +13,15 @@ const WEEKEND = [
   /week\-?end/i,
 ]
 
+
+// other patterns
+const USER_ALREADY_INTRODUCED = /suis|am|\'m/i
```

Comment:
Well looking at [API method](https://api.slack.com/methods), one could retrieve a [channel's history](https://api.slack.com/methods/channels.history) (to the last 1000 messages), but I not the full search :/